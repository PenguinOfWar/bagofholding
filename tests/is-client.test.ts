import isClient from '@/packages/is-client/is-client';

it('returns true when the environment is a browser and window is defined', () => {
  expect(isClient()).toBeTruthy();
  expect(isClient()).toMatchSnapshot();
});
