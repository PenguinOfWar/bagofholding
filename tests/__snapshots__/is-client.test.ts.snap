// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns true when the environment is a browser and window is defined 1`] = `true`;
